tests/E2E/test/campaigns/full/11_international/translation.js

```diff
@@ -1 +1,44 @@
-require('../../high/11_international/translation');
```

Comment:
Please avoid duplicating code.

Reply:
I delete it in this PR, will fix it in other one